PseudoLocalizer.Core.Tests/PseudoLocalizer.Core.Tests.csproj

```diff
@@ -1,11 +1,12 @@
-﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="16.0">
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>PseudoLocalizer.Core.Tests</RootNamespace>
-    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
+    <LangVersion>8</LangVersion>
```

Comment:
Please remove this. `latest` is specified in `Directory.Build.props`, so this shouldn't be overridden here.

Reply:
Done. I rebased the 3.0 stuff onto my branch to make the merge easier, hence why you're seeing it here - I had assumed that this line was added as part of that.